src/AvaloniaEdit/Search/SearchPanel.xaml

```diff
@@ -174,4 +176,9 @@
             </ControlTemplate>
         </Setter>
     </Style>
+
+  <Style Selector="Button">
```

Comment:
I think this style would apply to any button, you should constrain it to the above template

Reply:
To any button that is in SearchPanel.xaml so I thought it was okay to do

Reply:
Yeah fixed that